scalameta/tokenizers/shared/src/main/scala/scala/meta/internal/tokenizers/CharArrayReader.scala

```diff
@@ -26,10 +26,15 @@ class CharArrayReader(input: Input, dialect: Dialect, reporter: Reporter) extend
   import reporter._
 
   /** Is last character a unicode escape \\uxxxx? */
-  def isUnicodeEscape = charOffset == lastUnicodeOffset
+  var isUnicodeEscape = false
```

Comment:
It's hard for me to tell if this change is solid or not, but the tests pass so I'm happy to give this an LGTM 👍 and hope for the best. If there is a regression then our tests are not solid enough. wdyt @xeno-by ?

Reply:
Hey sorry folks I'm somewhat overwhelmed by meetings after coming back to work after ScalaDays. How urgent is this review? I'll need a bit of time to do it properly, because I don't remember these details off the top of my head.

Reply:
It's no probelem, thank you for replying! Personally speaking, this is not so urgent issue for me, so reviewing this PR carefully would be helpful.